packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -87,11 +100,7 @@ export class EVMAAWallet extends LoggerWrapper {
 
     public async signTypedData(params: TypedDataDefinition) {
         return this.logPerformance("SIGN_TYPED_DATA", async () => {
-            try {
-                return await this.kernelClient.signTypedData(params);
-            } catch (error) {
-                throw new Error(`Error signing typed data. If this error persists, please contact support.`);
```

Comment:
Dropping this try catch since smartAccountClient has duplicate error handling

Reply:
Since all the other function that are on the chopping block have other logic that could still throw errors, I'm not removing anything from them yet.